src/main/webapp/i18n/en/learningPath.json

```diff
@@ -11,6 +11,11 @@
             },
             "competencyGraphModal": {
                 "title": "Learning path competency relations"
+            },
+            "generation": {
+                "title": "Welcome to learning paths!",
+                "description": "Welcome to your learning path! This feature lets you learn at your own pace with a personalized journey tailored to your performance. It adapts dynamically, ensuring you always have the most relevant content and challenges. Start now and experience a customized path to success!",
+                "generateButton": "Start my learning path"
```

Comment:
The translations don't seem 100% consistent between German and English here.

For example:

German: Willkommen bei **deinem Lernpfad**
English: Welcome to **learning paths**!

in English, the "Welcome to your learning path" that I would expect in the title (based on the German translation) is in the description instead. 

Is there a reason for this?

Reply:
True I first did the English ones, then thought that the 1to1 translations don't really fit in German. But now I adapted the English ones again. 👍